code/modules/mapping/space_management/space_reservation.dm

```diff
@@ -0,0 +1,70 @@
+
+//Yes, they can only be square.
+//Yes, I'm sorry.
+/datum/turf_reservation
+	var/list/reserved_turfs = list()
+	var/top_left_coords[3]
+	var/bottom_right_coords[3]

```

Comment:
These are named incorrectly, (1, 1) is the bottom-left of the map.

Reply:
I believe this is supposed to be the coordinates of the top left and bottom right of the *reserved area* - this would be reserving exactly one square.

Reply:
@SpaceManiac what fludd said